r/DESCRIPTION

```diff
@@ -48,7 +48,7 @@ Suggests:
     rmarkdown,
     testthat,
     tibble
-LinkingTo: cpp11
+LinkingTo: cpp11 (>= 0.2.0)
```

Comment:
I believe we might actually require `>= 0.2.3`, looking at https://github.com/r-lib/cpp11/releases. I think that was the point at which we switched off of the vendored copy. @bkietz do you recall?

Reply:
I tested out all archived versions on CRAN, and 0.2.0 compiled. But it probably doesn't hurt to require the latest.

Reply:
Alright, if you tested and 0.2.0 works, that's ok with me